bundle/edu.gemini.spModel.io/src/main/scala/edu/gemini/spModel/io/impl/migration/to2015B/To2015B.scala

```diff
@@ -4,14 +4,17 @@ import java.util.UUID
 
```

Comment:
I turned a bunch of literals into constants here, so those changes are interleaved with the important bits. Sorry.